Projects/Server/Gumps/BaseGump.cs

```diff
@@ -0,0 +1,66 @@
+/*************************************************************************
+ * ModernUO                                                              *
+ * Copyright 2019-2023 - ModernUO Development Team                       *
+ * Email: hi@modernuo.com                                                *
+ * File: BaseGump.cs                                             *
+ *                                                                       *
+ * This program is free software: you can redistribute it and/or modify  *
+ * it under the terms of the GNU General Public License as published by  *
+ * the Free Software Foundation, either version 3 of the License, or     *
+ * (at your option) any later version.                                   *
+ *                                                                       *
+ * You should have received a copy of the GNU General Public License     *
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>. *
+ *************************************************************************/
+
+using Server.Network;
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Server.Gumps
+{
+    public abstract class BaseGump
```

Comment:
@Voxpire is worried about name clashing. Let's see if we can come up with a better name?